youtube_dl/extractor/parliamentliveuk.py

```diff
@@ -29,13 +30,13 @@ def _real_extract(self, url):
             'http://vodplayer.parliamentlive.tv/?mid=' + video_id, video_id)
         widget_config = self._parse_json(self._search_regex(
             r'kWidgetConfig\s*=\s*({.+});',
-            webpage, 'kaltura widget config'), video_id)
-        kaltura_url = 'kaltura:%s:%s' % (widget_config['wid'][1:], widget_config['entry_id'])
+            webpage, 'kaltura widget config', flags=re.DOTALL), video_id)
```

Comment:
Move flags into regex.

Reply:
I'm sorry, but I have no idea how to do this. I've searched through the code of other information extractors, but I have only found examples of how I did it, e.g. [beatport.py](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/beatport.py#L50), [imdb.py](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/imdb.py#L64) or [vimeo.py](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/extractor/vimeo.py#L534).

Reply:
`r'(?s)...`